hera_mc/geo_handling.py

```diff
@@ -313,33 +322,292 @@ def get_location(self, to_find, query_date, show_location=False, verbosity='m'):
+        """
+        from hera_mc import cm_hookup
+        hookup = cm_hookup.Hookup(self.session)
+
+        at_date = cm_utils._get_astropytime(at_date)
+        self.get_station_types()
+        station_conn = []
+        for k, stn_type in self.station_types.iteritems():
+            if (station_types_to_check == 'all' or k in station_types_to_check):
+                for stn in stn_type['Stations']:
+                    station_dict = self.get_fully_connected_location_at_date(stn=stn,
+                                                                             at_date=at_date,
+                                                                             hookup=hookup,
+                                                                             fc=None,
+                                                                             full_req=full_req)
+                    if len(station_dict.keys()) > 1:
+                        station_conn.append(station_dict)
+        return station_conn
+
+    def get_fully_connected_location_at_date(self, stn, at_date, hookup, fc=None,
+                                             full_req=part_connect.full_connection_parts_paper):
+        """
+        Returns a dict with the fully connected info on at_date.
+
+        The returned dict keys are:
+            'station_name': name of station (string, e.g. 'HH27')
+            'station_type': type of station (string, e.g. 'HH', 'PI', etc)
+            'tile': UTM tile name (string, e.g. '34J'
+            'datum': UTM datum (string, e.g. 'WGS84')
+            'easting': station UTM easting (float)
+            'northing': station UTM northing (float)
+            'longitude': station longitude (float)
+            'latitude': station latitude (float)
+            'elevation': station elevation (float)
+            'antenna_number': antenna number (integer)
+            'correlator_input_x': correlator input for x (East) pol (string e.g. 'DF7B4')
+            'correlator_input_y': correlator input for y (North) pol (string e.g. 'DF7B4')
+            'start_date': start of connection in gps seconds (long)
+            'stop_date': end of connection in gps seconds (long or None no end time)
+
+        Parameters
+        -----------
+        stn:  is the station information to check
+        at_date:  date to check for connections, must be an astropy.Time
+        hookup:  is to provide a hookup instance to not have to do it everytime
+        fc:  is the full hookup/revision, in case you already have it don't call it again
+        full_req:  list contining needed parts to be considered complete
+        """
+        if fc is None:
+            fc = cm_revisions.get_full_revision(stn, at_date, full_req, self.session)
+        station_dict = {}
+        if len(fc) == 1:
+            hu = fc[0].hookup
+            k0 = hu['hookup'].keys()[0]
+            ant_num = hu['hookup'][k0]['e'][0].downstream_part
+            ant_num = int(str(ant_num).split('A')[1])
```

Comment:
What's the format of the `ant_num` value? If it's just (e.g.) `A101`, this could just be `ant_num = int(ant_num[1:])`.